docker-compose.yml

```diff
@@ -91,9 +91,57 @@ services:
       test: ["CMD", "rabbitmq-diagnostics", "check_port_connectivity"]
       interval: 5s
       timeout: 30s
-      retries: 5
+      retries: 12
+    volumes:
+      - rabbitmq_data:/var/lib/rabbitmq
     networks:
       - fit
 
+  coach-consumer:
+    build:
+      context: .
+      dockerfile: coach.Dockerfile
+    command: ["python", "-m", "src.coach.wod_consumer"]
+    depends_on:
+      - rabbitmq
+      - coach-db
+      - monolith
+    volumes:
+      - ./src/coach:/app/src/coach
+    environment:
+      - RABBITMQ_HOST=rabbitmq
+      - RABBITMQ_DEFAULT_USER=rabbit
+      - RABBITMQ_DEFAULT_PASS=docker
+      - DATABASE_URL=postgresql://postgres:docker@coach-db:5432/coach-db
+      - MONOLITH_URL=http://monolith:5000
+      - FIT_API_KEY=some_very_secret_key
+    dns:
+      - 8.8.8.8
```

Comment:
As discussed, you didn't have to create a separate service to consume messages, you could have started the consumer on a separate thread but I like the cleverness of your solution 👍